site/_docs/history.md

```diff
@@ -54,18 +150,321 @@ other software versions as specified in gradle.properties.
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5768">CALCITE-5768</a>]
+  JDBC adapter should insert a subquery for a query with ORDER BY ordinal
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5676">CALCITE-5676</a>]
+  In JDBC DatabaseMetaData.getColumns, set DATA_TYPE and TYPE_NAME metadata values for MEASURE types
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5401">CALCITE-5401</a>]
+  Rule fired by HepPlanner can return Volcano's RelSubset
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5721">CALCITE-5721</a>]
+  Capture build scans on ge.apache.org to benefit from deep build insights
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5703">CALCITE-5703</a>]
+  Reduce amount of generated runtime code
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5717">CALCITE-5717</a>]
+  RelBuilder.project of literals on a single-row Aggregate should create a Values
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5697">CALCITE-5697</a>]
+  RelBuilder.convert does not match nullability if top of stack is a Project
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5839">CALCITE-5839</a>]
+  EnumerableInterpretable#StaticFieldDetector can overwrite its flag and return an incorrect result
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5479">CALCITE-5479</a>]
+  FamilyOperandTypeChecker is not readily composable in sequences
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5824">CALCITE-5824</a>]
+  Handle IndexCondition null pointQueryKey list in innodb
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5769">CALCITE-5769</a>]
+  Optimizing 'CAST(e AS t) IS NOT NULL' to 'e IS NOT NULL'
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5780">CALCITE-5780</a>]
+  Simplify '1 > x OR 1 <= x OR x IS NULL' to TRUE
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5708">CALCITE-5708</a>]
+  SUBSTRING validation error if any parameter is a NULL literal
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5816">CALCITE-5816</a>]
+  Only return left-hand table columns when validate LEFT SEMI JOIN query
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5798">CALCITE-5798</a>]
+  Improve simplification of '(x < y) IS NOT TRUE' when x and y are not nullable
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5810">CALCITE-5810</a>]
+  Prevent overflow in substring length computation
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5793">CALCITE-5793</a>]
+  JDBC adapter should use NULLS FIRST, NULLS LAST syntax for BigQuery
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5789">CALCITE-5789</a>]
+  Query with two nested subqueries where the inner-most references the outer-most table returns wrong result
+* [MINOR] Adjust nullableType and java doc for RexProgramBuilderBase
+* [MINOR] Assert can be replaced in `AbstractMaterializedViewRule#perform()`
```

Comment:
In general MINOR things should be excluded from release notes but not sure if in this case the commits are really minor. Better remove the `[MINOR]` prefix.

Reply:
Remove them.